wrappers/javascript/shared/src/types/IndyVdr.ts

```diff
@@ -84,7 +79,7 @@ export interface IndyVdr {
 
   buildGetTxnRequest(options: GetTransactionRequestOptions): number
 
-  buildGetValidatorInfoRequest(options: GetValidatorInfoRequestOptions): number
+  buildGetValidatorInfoRequest(options: GetValidatorInfoActionOptions): number
```

Comment:
Why did you rename it to actions?

Reply:
Because the getValidatorInfo should be run as action (If I understand the Python wrapper correctly):

https://github.com/hyperledger/indy-vdr/blob/d40a818dbee44e47476bc36a6f1a2249530a732f/wrappers/python/indy_vdr/pool.py#L70-L75